relay-server/src/services/processor.rs

```diff
@@ -2550,6 +2550,7 @@ impl EnvelopeProcessorService {
 
                 while let Some(bucket) = remaining.take() {
                     if let Some(next) = partition.insert(bucket, *scoping) {
+                        metric!(counter(RelayCounters::PartitionOverflow) += 1);
```

Comment:
Please, collect the count outside the loop and emit a single metric. (I'm not sure if the statsd proxy solves this indirectly, but it doesn't harm doing it)

The metric itself LGTM, but note that this metric may generate a very high volume and Relay already generates too many metrics. Once the decision is made, we should remove this metric.

Reply:
Also -- I haven't checked if we have a metric for this use case already

Reply:
Good point, will do!

Reply:
But +1 in keeping the count outside, doing a boolean is not the same as keeping the count.